cmd/kaniko-docker/main.go

```diff
@@ -146,9 +154,26 @@ func run(c *cli.Context) error {
 			EnableCache:   c.Bool("enable-cache"),
 			CacheRepo:     c.String("cache-repo"),
 			CacheTTL:      c.Int("cache-ttl"),
+			DigestFile:    defaultDigestFile,
 		},
 	}
-	return plugin.Exec()
+	err = plugin.Exec()
+	if err != nil {
+		return err
+	}
+
+	if c.String("artifact-file") != "" {
```

Comment:
This is a common code. Move it to kaniko.go file.

Reply:
done